src/NHSD.BuyingCatalogue.Organisations.Api/ViewModels/Organisations/UpdateOrganisationViewModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace NHSD.BuyingCatalogue.Organisations.Api.ViewModels.Organisations
+{
+    public sealed class UpdateOrganisationViewModel
```

Comment:
As above, this isn't actually a view model but there are currently inconsistencies and differences of opinion on this one so this is something we're planning to discuss tomorrow.

Reply:
Reply as above.